pkg/auth/authproviders/provider_impl.go

```diff
@@ -219,10 +219,15 @@ func (p *providerImpl) Validate(ctx context.Context, claims *tokens.Claims) erro
 		return errProviderDisabled
 	}
 
+	if err := validateTokenProviderUpdate(p.StorageView(), claims); err != nil {
+		return errors.Wrap(err, "provider has been updated")
```

Comment:
```suggestion
		return errors.Wrap(err, "tokens issued prior to provider update cannot be used")
```
When deciding what to write as error and wrapped message, I look at the following:
* how will the entire error chain look? ideally during a manual test
* what information the caller will not know?

Here we know what `validateTokenProviderUpdate()` checks for but `Validate()` callers will not know that. Btw, I assumed `validateTokenProviderUpdate()` does not mean to be generic auth provider validation but is limited to this very specific check.